tests/setup.sh

```diff
@@ -12,8 +12,19 @@ elif [ "$SUITE" = "phpunit-full" ];
 then
     export MOODLE_DOCKER_PHPUNIT_EXTERNAL_SERVICES=true
     initcmd="bin/moodle-docker-compose exec -T webserver php admin/tool/phpunit/cli/init.php"
-elif [ "$SUITE" = "app" ];
+elif [ "$SUITE" = "behat-app" ];
 then
+    git clone --branch v3.8.2 --depth 1 git://github.com/moodlehq/moodle-local_moodlemobileapp $HOME/moodle/local/moodlemobileapp
+
+    initcmd="bin/moodle-docker-compose exec -T webserver php admin/tool/behat/cli/init.php"
+elif [ "$SUITE" = "behat-app-development" ];
+then
+    git clone --branch v3.8.2 --depth 1 git://github.com/moodlehq/moodleapp $HOME/app
```

Comment:
And, for testing purposes, no matter it's not used normally... maybe we could also pass $APP_VERSION (just thinking that we may want to test multiple or, tomorrow... we can change the versions without touching the scripts.

Reply:
I couldn't do this before because of variables precedence (`MOODLE_APP_VERSION` took precedence over `MOODLE_DOCKER_APP_PATH`). I've changed the precedence to support this in the tests. It shouldn't be a problem outside of travis because the two variables are not supposed to be used together.

Reply:
Yeah, saw that if/elif precedence later. Good fix!